snapshots/manager.go

```diff
@@ -55,13 +61,29 @@ type Manager struct {
 }
 
 // NewManager creates a new manager.
-func NewManager(store *Store, target types.Snapshotter) *Manager {
+func NewManager(store *Store, multistore types.Snapshotter, extensions map[string]types.ExtensionSnapshotter) *Manager {
```

Comment:
Can we also add changelogs for all API-breaking changes?

Reply:
> Can we also add changelogs for all API-breaking changes?

should we add a new constructor to avoid break API, in this backport PR?

Reply:
I've added a new constructor, and added a improvement changelog.